src/components/show-edit-pod/show-edit-pod.stories.mdx

```diff
@@ -13,6 +14,10 @@ import Textbox from "../textbox";
 
 # ShowEditPod
 
+<DeprecationWarning>
+We've deprecated ShowEditPod. It's no longer available in the Product Design System or Figma library and we recommend that you do not use it in new products. See Related Components section for alternatives.
```

Comment:
What do you think about this small change to make the message more concise? This is really minor so I'm happy to accept how it is currently if you disagree :)

```suggestion
We've deprecated ShowEditPod. It's no longer available in the Design System and we recommend that you do not use it in new products. See Related Components section for alternatives.
```